guides/tutorials/codelabs/hello-deployment/index.md

```diff
@@ -276,22 +276,23 @@ our app.
 ![](create.png)
 
 
-### Create security group
+### Create firewall
 
-Now we will create a security group to allow access to our application. Spinnaker only allows you to
- attach ingress sources based on another security group, so we first must create a base security 
- group via the aws console to allow traffic on port 8080.
+Now we will create a firewall to allow access to our application. Spinnaker only allows you to
+ attach ingress sources based on another firewall, so we first must create a base security
+ group via the aws console (AWS Security Groups map to Firewalls in Spinnaker) to allow traffic
```

Comment:
Can you move the parenthetical phrase to the end of the sentence?

Reply:
Done.